apps/nextjs-website/src/components/molecules/QuestionsAndAnswers/QuestionsAndAnswers.tsx

```diff
@@ -0,0 +1,68 @@
+'use client';
+import React, { useState } from 'react';
+import SectionTitle from '../SectionTitle/SectionTitle';
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Box,
+  Button,
+  Typography,
+  useTheme,
+} from '@mui/material';
+import { Compress, Expand, ExpandMore } from '@mui/icons-material';
+import { useTranslations } from 'next-intl';
+import { QuestionsAndAnswers } from '@/lib/types/webinar';
+import BlocksRendererClient from '../BlocksRendererClient/BlocksRendererClient';
+
+export type QuestionsAndAnswersProps = {
+  readonly questions: QuestionsAndAnswers[];
+};
+
+const QuestionsAndAnswersComponent = ({
+  questions,
+}: QuestionsAndAnswersProps) => {
+  const theme = useTheme();
+  const t = useTranslations('webinar.webinarsSection.questionsAndAnswers');
+  const [showMore, toggleShowMore] = useState(false);
+  const questionsToShow = showMore ? 10 : 5;
```

Comment:
To avoid magic-number here i think is more readable to save `questionToShow` values in some const
```suggestion

const MIN_QUESTIONS_TO_SHOW = 5;
const MAX_QUESTIONS_TO_SHOW = 10;

export type QuestionsAndAnswersProps = {
  readonly questions: QuestionsAndAnswers[];
};

const QuestionsAndAnswersComponent = ({
  questions,
}: QuestionsAndAnswersProps) => {
  const theme = useTheme();
  const t = useTranslations('webinar.webinarsSection.questionsAndAnswers');
  const [showMore, toggleShowMore] = useState(false);
  const questionsToShow = showMore
    ? MAX_QUESTIONS_TO_SHOW
    : MIN_QUESTIONS_TO_SHOW;
```